src/gui/plugins/component_inspector/Vector3d.qml

```diff
@@ -35,46 +35,9 @@ Rectangle {
   // Horizontal margins
   property int margin: 5
 
-  // Maximum spinbox value
-  property double spinMax: 1000000
-
   // Units, defaults to meters.
   property string unit: model && model.unit != undefined ? model.unit : 'm'
```

Comment:
Let's remove this and move it below to where `gzUnit` is. Eg.
```javascript
gzUnit: model && model.unit != undefined ? model.unit : 'm'
```

Reply:
Fixed in a8c5a7d